pkg/constants/constants.go

```diff
@@ -25,6 +24,7 @@ const (
 	EtcdadmControllerProviderName           = "bootstrap-etcdadm-controller"
 	DefaultHttpsPort                        = "443"
 	DefaultWorkerNodeGroupName              = "md-0"
+	PodSubnetNodeMaskMaxDiff                = 16
```

Comment:
This constant doesn't really need to be here. Can we declare it at the site of use and unexport it.